lib/schedule.py

```diff
@@ -18,6 +19,12 @@ def get_seconds_into_day():
     return seconds_into_day
 
 
+def get_seconds_now():
+    now = datetime.datetime.now()
+    seconds_now = now.total_seconds()
+    return seconds_now
```

Comment:
`datetime` doesn't have a function `total_seconds()`. 

What do we want to get here? Seconds since epoch?

Reply:
If we want the time that a gradient was created by the user, maybe we want to store a `datetime` object on the gradient?